src/shared/dfsp-backend-requests.ts

```diff
@@ -167,10 +206,10 @@ export class DFSPBackendRequests extends HttpRequests {
     return this.post(this.validateAuthTokenPath, validateRequest)
   }
 
-  // validate ThirdpartyTransactionRequest
-  async validateThirdpartyTransactionRequest (
+  // validate ThirdpartyTransactionRequest and get context we will need to complete the transaction
```

Comment:
I'm fine with coupling the validate/lookup logic but you'll need to update the schema in `mojaloop-simulator` and update integration tests.

https://github.com/mojaloop/mojaloop-simulator/pull/97/files?file-filters%5B%5D=.yaml#diff-eb2a495bae6f23905b017eb7fea80e6920cb1c2356c450bc933ac38914bfb564R807

I personally don't mind if you just use `type: object` in `mojaloop-simulator` since I feel like we need to improve on our backend api and the request/response structures.

Reply:
Yep, I was thinking about this as well... Integration tests in this repo, or in the mojaloop-simulator repo?

Reply:
This repo.

Reply:
Ah I see. I forgot to look at the `pisp/master` branch on the mojaloop-simulator so I didn't follow your most recent changes